cmd/market_observer/main.go

```diff
@@ -84,10 +85,22 @@ func init() {
 
 func main() {
 	rateCron := market.InitRates(cache, rateProviders)
-	defer rateCron.Stop()
+	defer func() {
+		c := rateCron.Stop()
+		select {
+		case <-time.After(1 * time.Second):
```

Comment:
does this mean we will wait one second for the cron to gracefully complete? If so, how do you feel about moving this to a constant named for instance `gracefulShutdownTimeoutSeconds` such that it's a bit more clear?

Reply:
sure, added